src/main/scala/chiseltest/internal/TestEnvInterface.scala

```diff
@@ -56,10 +56,19 @@ trait TestEnvInterface {
     }
   }
 
+  protected def bigintToHex(x: BigInt): String = {
```

Comment:
Note that some viewers like GtkWave will pad leading zeros on hex strings to represent their widths.

Reply:
Is there a consensus on whether this should be done or not, especially within the Chisel ecosystem?

Reply:
Quantizing widths to multiples of the log of the radix isn't ideal, which IIRC was part of the rationale for moving away from doing things like inferring that `"h1".U` has a width of 4.

Reply:
Yeah, I can see that it would be a deal-breaker for specifying a literal, but it seems less clear that it's an obvious bad idea to pad zeros based on width for display purposes? That leading zero might represent more bits than there actually are, but I'm not sure if that edge case is worth it (or not?) to have an (imperfect) indicator of wire width?

Reply:
It's worth thinking about the implications for signed numbers.  If the width is inferred to be greater than actual, but the value isn't sign-extended to the new width, it'll appear to be a positive number.

Reply:
Ok, leaving as-is (not padded), if anyone really cares / has more information on general norms, we can revisit this later. Feel free to open an issue.

Some alternate options:
- Don't zero-pad (currently the case)
- Zero-pad hex only
- Zero-pad both hex and decimal
- Some other type of width indicator, perhaps displaying the data type and width, instead of leading zeros